scripts/post-process-generated-code.py

```
@@ -228,6 +228,54 @@ def fix_nullable_constructor_calls(content: str) -> Tuple[str, bool]:
+    if not properties:
+        return content, False
+
+    original = content
+    for prop in properties:
+        # Pascal-case the snake_case property name (is_confirmed -> IsConfirmed)
+        pascal = "".join(part.capitalize() for part in prop.split("_"))
+
+        # 1. Property declaration: public bool IsConfirmed { get; set; }
+        content = re.sub(
+            rf'(\[DataMember\(Name = "{re.escape(prop)}"[^\]]*\]\s*)public\s+bool\s+({pascal}\s*\{{\s*get;\s*set;\s*\}})',
+            rf'\1public bool? \2',
```

Ruff (0.15.12)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)